media-gfx/hydrus/hydrus-454.ebuild

```diff
@@ -0,0 +1,139 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
8 should work here, some changes in `src_install` that might be affected. I'll test and add it if it does.

Thanks!

Reply:
Somehow I didn't know EAPI 8 is already a thing at all.. Thank you.